akka-stream/src/main/scala/akka/stream/SslTlsOptions.scala

```diff
@@ -0,0 +1,255 @@
+/**
+ * Copyright (C) 2015-2016 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.stream
```

Comment:
I'm not sure I follow, because I could swear Akka Streams has a scaladsl and a javadsl package?